src/Combinatorics/Matroids/realization_space.jl

```diff
@@ -0,0 +1,989 @@
+
+
+mutable struct MatroidRealizationSpace
+    defining_ideal::Union{Ideal, NumFieldOrdIdl, Nothing}
+    inequations::Union{Vector{Oscar.RingElem},Nothing}
+    ambient_ring::Union{Oscar.MPolyRing, Ring, Nothing}
+    realization_matrix::Union{Oscar.MatElem,Nothing}
+    realizable::Union{Bool,Nothing} # Nothing if we're not sure yet if M is realizable.
+    F::AbstractAlgebra.Ring
+    char::Union{Int,Nothing}
+    q::Union{Int,Nothing}
+    one_realization::Bool
+end
+
+function Base.show(io::IO, RS::MatroidRealizationSpace)
+    if RS.realizable == false
+        if RS.char == nothing && RS.q == nothing && RS.F == ZZ
+            print(io, "The matroid is not realizable.")
+        else
+            print(io, "The matroid is not realizable over the specified field or characteristic.")
+        end
+    else
+        if RS.one_realization
+            println(io, "One realization is given by")
+        else
+            println(io, "The realizations are parametrized by")
+        end
+        # println isn't ideal as it prints the matrix as one big line
+        display(RS.realization_matrix)
```

Comment:
This is wrong as it ignores the `io` object.

Please also take a look at the "Printing" section of the OSCAR Developer documentation under https://docs.oscar-system.org

Reply:
I used display for the matrix as the other options `show` and `print`printed the matrix  in one big line.
Do you know which command to use to print a matrix nicely?

Reply:
Ah great thanks. Yes that works.
I changed that so this point is done.